### DISCUSSION TITLE
Remove padding-top from mantine-Stack-root

### DISCUSSION THREAD

**Discussion by amerfarooq** [mantinedev/mantine#3940] [Q&A] [answered]
I've noticed that when I use a Radio Group, there is a particular class `mantine-Stack-root` which has a `padding-top` of `8px`. I want to remove this padding but I can't find it anywhere in the Styles API of either the Radio or Stack components:

![image](https://user-images.githubusercontent.com/29232491/228313302-ce0c3383-4eef-456e-ba2f-790842600b0e.png)

![image](https://user-images.githubusercontent.com/29232491/228313356-92ab2994-d17f-4604-9819-159b183f798b.png)

**Comment by rtivital** [comment, marked as answer]
In 5.x you can target it with static selector: `<Radio.Group sx={{ '& .mantine-Stack-root': { paddingTop: 0 } }} />`. In 6.0 it should not be an issue.

**Reply by amerfarooq** [reply]
Thank you, that worked pretty well! Is there a way to generalize this across the whole theme or will I have to set this for every Radio.Group?

**Reply by rtivital** [reply]
You should be able to do that with Styles API on MantineProvider for Radio.Group – https://mantine.dev/styles/styles-api/#styles-api-with-mantineprovider
